starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +18,64 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(arr) {
+  var lg = 0;
+  var longest;
+  for (var i = 0; i < arr.length; i++) {
+    if (arr[i].length > lg) {
+      lg = arr[i].length;
+      longest = arr[i];
+    }
+ }
+ return longest;
+}
+
 // Calculating a Sum
 
 var numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+
+function sumArray(numbers) {
```

Comment:
Esta función y `averageNumbers` me da la sensación de que no deberían estar funcionando como deben, el reduce no te hace falta tenerlo dentro de un bucle for, te dejo una solución:

```
function sumArray(numbers) {
  return numbers.reduce(function(acc, current) {
    return acc + current;
  }, 0)
}
```
```
function averageNumbers(numbers) {
  return numbers.length > 0 ? sumArray(numbers) / numbers.length : undefined;
}
```

Como ves, en `averageNumber` podemos reutilizar `sumArray`